index.bs

```diff
@@ -3384,6 +3384,19 @@ To <dfn>locate nodes using inner text</dfn> with given |context nodes|,
 
 1. For each |context node| in |context nodes|:
 
+  1. If |context node| implements {{Document}} or {{DocumentFragment}}:
```

Comment:
Note: so we traverse in-depth only html elements, documents and document fragments, while ignoring any other non-element nodes.